reference_data/management/tests/update_hpo_2_3_tests.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import unicode_literals
```

Comment:
Maybe because of most of lines have been changed for removing the leading `u`.
Besides the leading `u`, the changes include:
1. changed the data file writing mode to binary mode and add encoding before writing (line #144 and #157)
2. change exception message to str(exception) in line #161.